pkgs/top-level/all-packages.nix

```diff
@@ -38035,7 +38035,7 @@ with pkgs;
 
   gav = callPackage ../games/gav { };
 
-  gcs = callPackage ../games/gcs { };
+  gcs = callPackage ../games/gcs { go = go_1_21; };
```

Comment:
pass `buildGoModule = buildGoModule121` this is more consistent with how go version changes are usually handled